### DISCUSSION TITLE
How do I add a client route?

### DISCUSSION THREAD

**Discussion by agamm** [vercel/next.js#13229] [Help]
I want to have a few pages generated statically and a route at /app which is a client-side react app (no need for SEO).  
In Gatsby there is [gatsby-plugin-create-client-paths](https://www.gatsbyjs.org/packages/gatsby-plugin-create-client-paths/)

Is there an equivilant / a solution?
Bonus points if there isn't any noticable page reload

**Comment by VivekNeel** [comment]
Using a Link component from next/link. Ref : https://nextjs.org/docs/api-reference/next/link

**Comment by agamm** [comment]
@VivekNeel what do you mean? I already am using it but how can I connect it to a react app ?
```
 <p className="description">
          <Link href="/clientside"><a>Client side app (react)!</a></Link>
  </p>
```
`pages/clientside` is basically a create-react-app I have (client side app, no need to statically generate).
